src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="d-flex flex-column justify-content-center ">
+      <section className="align-self-center text-center">
```

Comment:
I don't think we need to use two sections here since we only have one single section, which is "take a quiz, here are the links to the quiz"

Reply:
Fixed it.